install/kubernetes/helm/istio/charts/pilot/templates/gateway.yaml

```diff
@@ -2,7 +2,6 @@ apiVersion: networking.istio.io/v1alpha3
 kind: Gateway
 metadata:
   name: istio-autogenerated-k8s-ingress
-  namespace: istio-system
```

Comment:
this one certainly needs parameterization.

Reply:
And it needs the same param as on the other charts.